### PR TITLE
add modified_survey_point compare function

### PR DESCRIPTION
**Pull request by willemarcel**
Detects `man_made=survey_point` deleted, moved or that lost its main tag

### CODE REVIEW

File: comparators/modified_survey_point.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = modifiedSurveyPoint;
+
+function modifiedSurveyPoint(newVersion, oldVersion) {
+  if (
+    !newVersion.deleted &&
+    newVersion.properties['osm:version'] !== 1 &&
```
Comment (amishas157) [inline comparators/modified_survey_point.js]: Hi @willemarcel , we have a basic fixture https://github.com/mapbox/osm-compare/blob/master/tests/basicFixture.json which always gets tested against all the compare functions, in that it also have fixtures where either of oldversion/newversion is null. Therefore we always make sure the following checks before using them:
- oldVersion
- newVersion
- oldVersion.properties
- newVersion.properties
- oldVersion.geometry
- newVersion.geometry

File: tests/fixtures/modified_survey_point.json [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                        89,
+                        89
+                    ]
+                }
+            },
+            "expectedResult": {
+                "result:modified_survey_point": true
+            }
+        }
+    ]
```
Comment (amishas157) [inline tests/fixtures/modified_survey_point.json]: The last test case seems like same as the test case just before it

**Comment by amishas157** [comment]
Hi @willemarcel 👋 , this PR looks good to me . Thanks for the contributions. 🙇‍♀️ 

> Detects man_made=survey_point deleted, moved or that lost its main tag

By main tag did you mean the name tag or also some other tags also?

File: tests/fixtures/modified_survey_point.json (hunk 2 above)
Comment (willemarcel) [reply tests/fixtures/modified_survey_point.json]: In one case the tag `man_made=survey_point` was changed to `historic=monument`. In the other, it changed to `man_made=obelisk`

**Pull request merged**